tests/PerformanceTest/IniReadingBench.php

```diff
@@ -1,57 +1,59 @@
 <?php
 
-namespace Matomo\Tests\Ini\PerformanceTest;
+namespace Matomo\Ini\Tests\PerformanceTest;
 
-use Athletic\AthleticEvent;
 use Matomo\Ini\IniReader;
+use PhpBench\Benchmark\Metadata\Annotations\Iterations;
 
-class IniReadingEvent extends AthleticEvent
+/**
+ * Class IniReadingBench
+ */
+class IniReadingBench
 {
+
     /**
      * @var IniReader
      */
     private $reader;
 
+    /**
+     * @var string
+     */
     private $ini;
 
-    protected function classSetUp()
-    {
-        if (extension_loaded('xdebug')) {
-            throw new \Exception('Xdebug should be disabled to run the performance tests');
-        }
-    }
-
-    protected function setUp()
-    {
-        $this->reader = new IniReader();
-        $this->createIniString();
-    }
-
     /**
-     * @iterations 10000
+     * @BeforeMethods("setUp", "createIniString")
+     * @Iterations(5)
      */
-    public function native()
+    public function benchNative()
     {
         parse_ini_string($this->ini, true);
     }
 
     /**
-     * @iterations 10000
+     * @BeforeMethods("setUp", "createIniString")
+     * @Iterations(5)
```

Comment:
Is there any reason for lowering the iterations count for those tests?

Reply:
No, sorry, I just had lowered that to speed up while switching to phpbench. Fixed in f753034c.